AppCenterAnalytics/AppCenterAnalytics/MSACAnalytics.h

```diff
@@ -197,6 +197,18 @@ NS_SWIFT_NAME(Analytics)
  */
 + (void)resume;
 
+/**
+ Start session manually.
+ *
+ */
++ (void)startSession;
+
+/**
+ * Setting the state of manually session generation process.
+ *
```

Comment:
extra `*`

Reply:
`Setting the state of manually session generation process.` -> `Enable manual session tracker.`